kolibri/core/assets/src/views/HorizontalNavBarWithOverflowMenu.vue

```diff
@@ -120,7 +120,6 @@
   @import '~kolibri-design-system/lib/styles/definitions';
 
   .menu-icon {
-    position: absolute;
     right: 4px;
```

Comment:
With `position` removed, `right` will have no effect, so the whole `.menu-icon` can be removed from here as well as its references in the markup

Reply:
oh, what a good point 😄 thanks, Misha!